resources/assets/js/snipeit.js

```diff
@@ -191,17 +191,15 @@ $(document).ready(function () {
      * Select2
      */
 
-     var iOS = /iPhone|iPad|iPod/.test(navigator.userAgent)  && !window.MSStream;
-     if(!iOS)
-     {
+
         // Vue collision: Avoid overriding a vue select2 instance
         // by checking to see if the item has already been select2'd.
```

Comment:
It's not necessary but can we remove this comment since it is no longer relevant por favor?

Reply:
done 👍